v2/config/rbac/kustomization.yaml

```diff
@@ -8,9 +8,9 @@ resources:
   # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
   # which protects your /metrics endpoint.
   - auth_proxy_service.yaml
-  - auth_proxy_role.yaml
-  - auth_proxy_role_binding.yaml
-  - classic/role_binding.yaml
+  # - auth_proxy_role.yaml
+  # - auth_proxy_role_binding.yaml
+  # - classic/role_binding.yaml
```

Comment:
not sure if these were commented by mistake. They are still in deployer/v2/config/rbac/kustomization.yaml

Reply:
classic role binding was moved
auth proxy was not a correct configuration, nor used